api/src/main/resources/api.yaml

```diff
@@ -1223,4 +1281,75 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/AdminPayment'
-          uniqueItems: true
\ No newline at end of file
+          uniqueItems: true
+    ListHouseHistoryResponse:
+      type: object
+      properties:
+        histories:
```

Comment:
`histories` - is this name appropriate?
How about `houseRentalHistory`, `rentalHistory`?